### DISCUSSION TITLE
Effect of First Component on Downstream Analysis

### DISCUSSION THREAD

**Discussion by galib36** [stuart-lab/signac#371] [Q&A] [answered]
Thank you for making `signac` available to the research community. As I was using the Human PBMC Vignette, I found that you do not consider the first LSI component in the downstream analysis as it has high correlation with the sequencing depth. I tried both including the first component and also discarding it. It looks like it does not make much impact on clustering and on tSNE plots. I was wondering whether you have also seen similar result? In that case, how much impact will it have in downstream analysis if I __keep__ the first LSI component?

**Comment by timoast** [comment, marked as answer]
In practice, retaining the first component may not always have a big effect on the clustering / UMAP. However, I have seen cases where retaining the first component can drive separation between groups of cells based on sequencing depth. If the first component is highly correlated with total counts I don't really see any reason to include it in the analysis.
